server/src/main/java/com/example/server/brave/BraveOtlpConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.example.server.brave;
+
+import brave.Tags;
+import brave.handler.MutableSpan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import zipkin2.reporter.BytesEncoder;
+import zipkin2.reporter.otel.brave.OtlpProtoV1Encoder;
+
+// tag::encoder[]
+@Configuration(proxyBeanMethods = false)
+public class BraveOtlpConfig {
+
+  @Bean
+  BytesEncoder<MutableSpan> otlpMutableSpanBytesEncoder() {
+    return new OtlpProtoV1Encoder(Tags.ERROR);
```

Comment:
The way to build the `OtlpProtoV1Encoder` has been changed to builter-pattern via https://github.com/openzipkin-contrib/zipkin-otel/pull/18

```suggestion
    return OtlpProtoV1Encoder.create();
```

And you can now also send resource attributes like [Spring Boot does for the Otel SDK](https://github.com/spring-projects/spring-boot/blob/aba22547ee56606c815ecb1ef57cee82e9005ed9/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/opentelemetry/OpenTelemetryAutoConfiguration.java#L74-L91).